system-tests/test/pause_headed_exit_spec.ts

```diff
@@ -0,0 +1,51 @@
+import systemTests from '../lib/system-tests'
+
+describe('cy.pause() works with --headed and --no-exit', () => {
```

Comment:
```suggestion
describe('cy.pause() in run mode', () => {
```
Might be worth generalizing the describe & test file name given this testing these various permutations.